### ISSUE TITLE
Converting speech_embedding hub module to tflite results in `Encountered unresolved custom op: TensorArrayV3.Node`

### ISSUE THREAD

**Issue by BryanWBear** [tensorflow/tensorflow#45864]
**System information**
- Colab default settings
- TF version: 2.4.0

**Describe the current behavior**
Converting speech_embedding module ''https://tfhub.dev/google/speech_embedding/1'' to tflite results in:

`RuntimeError: Encountered unresolved custom op: TensorArrayV3.Node number 2 (TensorArrayV3) failed to prepare.`

during inference.

**Standalone code to reproduce the issue**

```
HUB_URL = 'https://tfhub.dev/google/speech_embedding/1'
TEST_PATH = '.'

embedding_layer = hub.KerasLayer(HUB_URL, input_shape=(16000,), trainable=False)

model = tf.keras.Sequential([
    embedding_layer
])

model.save(TEST_PATH)
converter = tf.lite.TFLiteConverter.from_saved_model(TEST_PATH)


converter.allow_custom_ops = True
tflite_model = converter.convert()
tflite_model_file = 'converted_model.tflite'

with open(tflite_model_file, "wb") as f:
  f.write(tflite_model)

interpreter = tf.lite.Interpreter(model_path=tflite_model_file)
interpreter.allocate_tensors()
```

**Output**

```---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-16-5f54d02787e4> in <module>()
      1 interpreter = tf.lite.Interpreter(model_path=tflite_model_file)
----> 2 interpreter.allocate_tensors()
      3 
      4 input_index = interpreter.get_input_details()[0]["index"]
      5 output_index = interpreter.get_output_details()[0]["index"]

/usr/local/lib/python3.6/dist-packages/tensorflow/lite/python/interpreter.py in allocate_tensors(self)
    257   def allocate_tensors(self):
    258     self._ensure_safe()
--> 259     return self._interpreter.AllocateTensors()
    260 
    261   def _safe_to_run(self):

RuntimeError: Encountered unresolved custom op: TensorArrayV3.Node number 2 (TensorArrayV3) failed to prepare.```



I am aware that STFT may not be supported by tflite yet. If this is the issue, is there a quick workaround?

Thanks,
Bryan

**Comment by abattery** [comment]
Sorry for encountering this issue on my side. TensorArray is not supported via TensorFlow Lite currently.  We are working on supporting them in the future.

**Comment by mohdhamzahkhan** [comment]
I am trying to perform transfer learning using tf hub speech embedding . I was able to train it in tf 1.15 but to convert it to tflite 
I used tf 2.4 as tf 1.15 was giving errors. I was successfully able to  convert the model but I am not able to make predictions
for tf 2.4. my error is
  RuntimeError: Container __per_step_0 does not exist. (Could not find resource: __per_step_0/_tensor_arraysTensorArrayV3_0)
	 (while executing 'TensorArrayScatterV3' via Eager)Node number 82 (TfLiteFlexDelegate) failed to invoke.

I also tried using tf-nightly where also I encountered error 
RuntimeError: Tensor: tf.TensorArrayV3(56) buffer size mismatch 64(1) != 8(2)Node number 82 (TfLiteFlexDelegate) failed to invoke.

**Comment by sonal-511** [comment]
I am trying to perform transfer learning using tf hub speech embedding . I was able to train it in tf 1.15 but to convert it to tflite
I used tf 2.4 as tf 1.15 was giving errors. I was successfully able to convert the model but I am not able to make predictions
for tf 2.4. my error is
RuntimeError: Container __per_step_0 does not exist. (Could not find resource: __per_step_0/_tensor_arraysTensorArrayV3_0)
(while executing 'TensorArrayScatterV3' via Eager)Node number 82 (TfLiteFlexDelegate) failed to invoke.

I also tried using tf-nightly where also I encountered error
RuntimeError: Tensor: tf.TensorArrayV3(56) buffer size mismatch 64(1) != 8(2)Node number 82 (TfLiteFlexDelegate) failed to invoke.
